src/parser/whamm.pest

```diff
@@ -35,9 +35,12 @@ PROBE_SPEC = ${
     | PROBE_ID? ~ ":" ~ PROBE_ID?
     | PROBE_ID
 }
+//disallowed IDs should have the full list of reserved names like return
 
-ID = @{ (ASCII_ALPHA | "_")+ ~ ( ASCII_DIGIT | (ASCII_ALPHA | "_")+ )* }
-
+DISALLOWED_ID = _{ "return" | "if" | "while" | "else" }
+ID = @{ (!DISALLOWED_ID ~ (ASCII_ALPHA | "_")+ ~ ( ASCII_DIGIT | (ASCII_ALPHA | "_")+ )*) |
+        (DISALLOWED_ID ~ (ASCII_ALPHA | "_" | ASCII_DIGIT)+)
```

Comment:
You wouldn't need the `DISALLOWED_ID` in this line right? All we need is the `!DISALLOWED_ID` as a check.

Like this:
```
ID = @{ !DISALLOWED_ID ~ (ASCII_ALPHA | "_")+ ~ ( ASCII_DIGIT | (ASCII_ALPHA | "_")+ )* }
```

Reply:
This second line is what allows users to define variables like "return_value" as a valid name; without it, that would not be allowed